easy-ui-react/src/SplitToggleCard/SplitToggleCard.stories.tsx

```diff
@@ -0,0 +1,107 @@
+import { Meta, StoryObj } from "@storybook/react";
+import React, { useState, useId } from "react";
+import { SplitToggleCard, SplitToggleCardProps } from "./SplitToggleCard";
+import { Text } from "../Text";
+import { VerticalStack } from "../VerticalStack";
+
+type Story = StoryObj<typeof SplitToggleCard>;
+
+const meta: Meta<typeof SplitToggleCard> = {
+  title: "Components/SplitToggleCard",
+  component: SplitToggleCard,
+  args: {
+    isDisabled: false,
+    isReadOnly: false,
+  },
+  parameters: {
+    controls: {
+      exclude: ["onChange"],
+    },
+  },
+};
+
+const Template = (args: SplitToggleCardProps) => {
+  const id = useId();
+  const status =
+    args.isSelected || args.defaultSelected ? "Enabled" : "Disabled";
+  return (
+    <SplitToggleCard {...args} aria-labelledby={id}>
+      <VerticalStack gap="1">
+        <Text variant="subtitle2" color="primary.800">
```

Comment:
**issue**: this should have the id attached, `id={id}`, so that the `aria-labelledby` is connected to this text